test.py

```diff
@@ -1,4 +1,5 @@
 from test1 import func
```

Comment:
Add test cases to verify the functionality of the imported function `func` with different inputs to cover different scenarios and edge cases.

```suggestion
assert func(0) == expected_value_for_0, "Test case for 0 failed"
```